src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAO.scala

```diff
@@ -337,11 +343,18 @@ trait GraphDAO {
 
     val parameters = ctor.asMethod.paramLists.head.map { paramSymbol =>
       val paramName = paramSymbol.name.decodedName.toString.trim
+
```

Comment:
its kind of ugly that tryLoad is defined inside a map, but i didnt want to have to pass paramName, classT and vertex to that exception.  the other option was implicit arguments, but those would require paramName, classT and vertex to be implicit